File: LLM-Translations/GPT-4/avatar/Python/Go/atcoder_ABC119_B.go

package main

import (
	"fmt"
	"strings"
	"strconv"
)

func main() {
	var N int
	var val string
	var total float32 = 0.0
	
	fmt.Scan(&N)
	
	for i := 0; i < N; i++ {
		fmt.Scan(&val)
		if strings.Contains(val, "JPY") {
			val = strings.Trim(val, "JPY")
			amount, _ := strconv.ParseFloat(val, 32)
			total += float32(amount)
		} else if strings.Contains(val, "BTC") {
			val = strings.Trim(val, "BTC")
			amount, _ := strconv.ParseFloat(val, 32)
			total += float32(amount) * 380000.0
		}
	}
	
	fmt.Println(total)
}
// End of Code

File: LLM-Translations/GPT-4/avatar/Python/Java/atcoder_ABC119_B.java
import java.util.Scanner;

public class atcoder_ABC119_B {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        double[] x = new double[N];
        double total = 0;

        for (int i = 0; i < N; i++) {
            String input = sc.next();
            if (input.contains("JPY")) {
                x[i] = Double.parseDouble(input.replace("JPY", "")) * 1.0;
            } else if (input.contains("BTC")) {
                x[i] = Double.parseDouble(input.replace("BTC", "")) * 380000.0;
            }
            total += x[i];
        }
        System.out.println(total);
    }
}
// End of Code
